e2e/specs/st_file_uploader.spec.js

```diff
@@ -127,7 +127,7 @@ describe("st.file_uploader", () => {
 
         cy.getIndexed(
           "[data-testid='stMarkdownContainer']",
-          uploaderIndex
+          1
         ).should("contain.text", "True");
```

Comment:
A `WidgetLabel` uses `StreamlitMarkdown` now, `stMarkdownContainer` indices needed to be updated for these tests